src/client/Microsoft.Identity.Client.Broker/BrokerExtension.cs

```diff
@@ -23,10 +26,30 @@ public static class BrokerExtension
         /// If a broker does not exist or cannot be used, MSAL will fallback to a browser.
         /// Make sure browser auth is enabled (e.g. if using system browser, register the "http://localhost" redirect URI, etc.)
         /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("Use WithBroker(BrokerOptions) instead. This API can be found in Microsoft.Identity.Client.Broker package. See https://aka.ms/msal-net-wam for details.", false)]
```

Comment:
please have a work item to track doc updates after release

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4015